spec.bs

```diff
@@ -1230,11 +1352,22 @@ The <dfn for=Navigator method>updateAdInterestGroups()</dfn> method steps are:
     Implementations may consider loading only a portion of these interest groups at a time to avoid
     issuing too many requests at once.

```

Comment:
```suggestion

    Note: Implementations may consider loading only a portion of these interest groups at a time to avoid
    issuing too many requests at once.

```